### PR TITLE
fix(kit): `Prompt` wrap buttons to the new line if they do not fit

### LINKED ISSUE #4439: 🐞 - PromptDialog: buttons don't look good on mobile devices.
### Playground Link

-

### Description

When I open PromptDialog with long buttons text, it looks like that 
image

### Angular version

11

### Taiga UI version

2

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #4439

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/prompt/examples/import/service.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+```ts
+import {TUI_PROMPT} from './prompt.component';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/prompt/examples/import/service.md:2]:
```suggestion
import {TUI_PROMPT} from '@taiga-ui/kit';
```
?

File: projects/kit/components/prompt/prompt.style.less [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-    margin-bottom: 1.25rem;
+    margin-bottom: 0.875rem;
 }
 
 .t-buttons {
     display: flex;
     justify-content: flex-end;
-    margin-top: 0.75rem;
-    width: 100%;
+    flex-wrap: wrap;
```
Comment (nsbarsukov) [inline projects/kit/components/prompt/prompt.style.less:10]: Could you add Cypress screenshot test for this case?

https://taiga-ui--pr4443-prompt-orwo26dc.web.app/components/prompt/API?no=Very%20Long%20long%20text&yes=Long%20long%20text

File: projects/demo/src/modules/components/prompt/examples/1/index.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        @Inject(TuiDialogService) private readonly dialogs: TuiDialogService,
+        @Inject(TuiAlertService) private readonly alerts: TuiAlertService,
+    ) {}
+
+    onClick(): void {
+        const data: TuiPromptData = {
+            content:
+                'This is <code>PolymorpheusContent</code> so it can be template too!',
+            yes: 'That is great!',
+            no: 'Who cares?',
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/prompt/examples/1/index.ts:25]: I am not sure that `prompt` is appropriate naming.

See description of native [window.prompt](https://developer.mozilla.org/ru/docs/Web/API/Window/prompt):
> window.prompt() instructs the browser to display a dialog with an optional message prompting the user **to input some text**, and to wait until the user either submits the text or cancels the dialog.

Do you mean `TUI_CONFIRM` ?
See description of native [window. confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm):

> window.confirm() instructs the browser to display a dialog with an optional message, and to wait until the user either **confirms or cancels** the dialog.

Comment (nsbarsukov) [reply projects/demo/src/modules/components/prompt/examples/1/index.ts:25]: Oops, this component already exists ...

Comment (nsbarsukov) [reply projects/demo/src/modules/components/prompt/examples/1/index.ts:25]: Should we rename it in future major release ?

**Comment by splincode** [comment]
is it expected?

image

**Comment by waterplea** [comment]
> is it expected?
> 
> image
> 

A necessary evil.

File: projects/demo/src/modules/components/prompt/examples/1/index.ts (hunk 3 above)
Comment (waterplea) [reply projects/demo/src/modules/components/prompt/examples/1/index.ts:25]: Sure, added a TODO

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**